source/github-helpers/index.ts

```diff
@@ -116,3 +117,11 @@ export async function isPermalink(): Promise<boolean> {
 export function isNotRefinedGitHubRepo(): boolean {
 	return !location.pathname.startsWith('/sindresorhus/refined-github/');
 }
+
+const settings = optionsStorage.getAll();
+export async function logQuery(message: string): Promise<void> {
```

Comment:
I meant this function 

https://github.com/sindresorhus/refined-github/blob/ac8de6bd0ef0a6016732a69556e4eb9c014cfd38/source/features/index.tsx#L46


> 1. move this out of here and **into features/index**
> 2. conform its style to the **existing logging function**
> 3.  **make the type signature conform to console.log**, not just a single string parameter

Reply:
Also if the option is logHTTP then the function should not be logQuery

Reply:
Also https://github.com/sindresorhus/refined-github/blob/ac8de6bd0ef0a6016732a69556e4eb9c014cfd38/source/features/index.tsx#L103-L104